lib/widgets/about_zulip.dart

```diff
@@ -41,9 +34,10 @@ class _AboutZulipPageState extends State<AboutZulipPage> {
             child: ConstrainedBox(
               constraints: const BoxConstraints(maxWidth: 400),
               child: Column(mainAxisAlignment: MainAxisAlignment.center, children: [
-                ListTile(
-                  title: Text(zulipLocalizations.aboutPageAppVersion),
-                  subtitle: Text(_packageInfo?.version ?? '(…)')),
+                if (appVersion != null)
+                  ListTile(
+                    title: Text(zulipLocalizations.aboutPageAppVersion),
+                    subtitle: Text(appVersion)),
```

Comment:
This is a behavior change, so let's leave it out of a pure-refactor `[nfc]` commit. Before, when the app version is undetermined, we'd show a `ListTile` with subtitle "(…)". Now, in that case, the whole `ListTile` is omitted.

Reply:
removed this behavior change